packages/cactus-plugin-ledger-connector-polkadot/src/test/typescript/integration/deploy-ink-contract.test.ts

```diff
@@ -0,0 +1,89 @@
+/*
```

Comment:
@RafaelAPB Are these commented out because they are not needed at all to achieve sufficient coverage or some other reason?

Reply:
Tests are not currently passing because the smart contracts are not being generated automatically from the source, and thus we cannot run them. Is this an impeditive issue?